packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.jsx

```diff
@@ -0,0 +1,170 @@
+   */
+  applicationContent: PropTypes.node,
+  /**
+   * Age of the person.
+   */
+  age: PropTypes.string,
+  /**
+   * The persons date of birth
+   */
+  dateOfBirth: PropTypes.string,
+  /**
+   * The person's deceased date. Will display the banner as deceased if this value is provided
+   */
+  deceasedDate: PropTypes.string,
+  /**
+   * Gender of the Person
+   */
+  gender: PropTypes.string,
+  /**
+   * The persons gestational age.
+   */
+  gestationalAge: PropTypes.string,
+  /**
+   * Additional key value identifiers of a person's demographic information
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  identifiers: PropTypes.object,
+  /**
+   * @private
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
+   */
+  intl: intlShape.isRequired,
+  /**
+   * Full Name of the person
+   */
+  personName: PropTypes.string,
+  /**
+   * URL of Photo to display in the banner
+   */
+  photo: PropTypes.string,
+  /**
+   * The column layout in which to present the displays.
+   */
+  postMenstrualAge: PropTypes.string,
+  /**
+   * The persons preferred first name if they have one.
+   */
+  preferredFirstName: PropTypes.string,
+  /**
+   * Callback function triggered when info button is clicked.
+   */
+  infoButtonOnClick: PropTypes.func.isRequired,
+  /**
+   * Callback function triggered when selectable-tile is clicked.
+   */
+  selectableTileOnClick: PropTypes.func.isRequired,
+  /**
+   * Id for Info Button.
+   */
+  infoButtonId: PropTypes.string,
+  /**
+  * Id for Selectable Tile.
+  */
+  selectableTileId: PropTypes.string,
+};
+
+const defaultProps = {
+  applicationContent: null,
+  age: undefined,
+  dateOfBirth: undefined,
+  deceasedDate: null,
+  gender: undefined,
+  gestationalAge: null,
+  identifiers: {},
+  personName: undefined,
+  photo: undefined,
+  postMenstrualAge: null,
+  preferredFirstName: null,
+  infoButtonId: undefined,
+  selectableTileId: undefined,
+};
+
+const SelectableDemographicsBanner = ({
+  age,
+  dateOfBirth,
+  gender,
+  intl,
+  personName,
+  infoButtonOnClick,
+  selectableTileOnClick,
+  infoButtonId,
+  selectableTileId,
+  ...customProps
+}) => {
+  const noDataProvided = intl.formatMessage({ id: 'Terra.demographicsBanner.noDataProvided' });
+
+  const handleKeyDown = (event) => {
+    console.log('keyspace', KEY_SPACE);
+    console.log('keyspace', KEY_RETURN);
+    if (event.keyCode === KEY_SPACE || event.keyCode === KEY_RETURN) {
+      event.preventDefault();
+    }
+  };
+
+  const handleKeyUp = (event) => {
+    if ((event.keyCode === KEY_SPACE || event.keyCode === KEY_RETURN) && selectableTileOnClick) {
+      selectableTileOnClick();
+    }
+  };
+
+  const handleOnClick = () => {
+    if (selectableTileOnClick) {
+      selectableTileOnClick();
+    }
+  };
+
+  const DemographicsBannerSelectableTile = (
+    <DemographicsBannerDisplay
+      {...customProps}
```

Comment:
Custom props should always be spread on the root element.

Reply:
[updated](https://github.com/cerner/terra-core/pull/2856/files#diff-75915ebe8c2d0aabab6be6449bd17b56)

Reply:
In Selectable-demographics banner root element is just background `div` . so spreading customProps on Background-tile would be of no use I think. hence having it on BannerDisplay so that any props related to banner can be used by BannerDisplay and also avoids explicit listing of props here.

Reply:
It is expected that custom props be spread on the outer-most div. This is the precedent that we've set in our other components. If we want consumers to be able to apply custom attributes to a specific div within the component we add a named prop for that, but customProps would still be spread on the outer-most div.

Reply:
updated by ben with this [commit](https://github.com/cerner/terra-core/pull/2856/commits/fa099b0a16ef38530aac8da39595d64acb66d185)